src/ErrorLogger/customStyles.js

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components';
+
+export const CustomStylesWrapper = styled.div`
+.errorHeader{
+};
+.errorDisplay{
+};
+.errorBarArrow{
+};
+.errorComponent{
+};
+.errorFile{
+};
+.errorArrowDivExpanded{
+};
+.errorArrowDivCollapsed{
+};
+.errorType{
+};
+.errorShortMessage{
+};
+.errorFullMessage{
+};
+`
```

Comment:
Can you explain more why these custom style wrappers help / what they do?

Reply:
When I first added them they meant sense to me as it is one of the best way for **me** to added custom CSS style to elements. But now it doesn't seem required at all since the user will be adding his/her own wrapper around the component or style document. Sorry for the inconvenience, I'll be removing it!